src/components/templates/BottomSheetRemoveAddressFromCollection/index.tsx

```diff
@@ -39,7 +40,10 @@ export const BottomSheetRemoveAddressFromCollection = forwardRef<
           fontSize={14}
           color={COLORS.smokyBlack}
         >
-          Remove this address from {collection ? collection.name : 'collection'}
+          Remove this address from{' '}
+          {collection
+            ? StringUtils.formatAddress(collection.name, 10, 0)
```

Comment:
what do you think, will 10 characters would be enough here? or should we do more/less?